pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -46,19 +46,91 @@ const (
 	// RbacConfigName is the name of the RbacConfig custom resource that controls the RBAC behavior.
 	RbacConfigName = "rbac-config"
 
-	sourceIP        = "source.ip"
-	sourceService   = "source.service"
-	destinationIP   = "destination.ip"
```

Comment:
rename(ex source.ip->ip) means existing user facing configurations for mixer rbac need to be updated before applied to local rbac, is that expected ?

Reply:
how could a user specify attribute directly in the ServiceRole and ServiceRoleBinding? I think the attribute name only appears in the Authorization template which won't be needed anyway in local RBAC.